### PR TITLE
wait for context view to hide so screen reader detects focus change, add toolbar

### PR DESCRIPTION
**Pull request by meganrogge**
cc @rperez030 this now works and feels nice

still some context key issues to fix which cause the actions to become disabled. that can happen in a separate PR and I have deferred #189981 as it's not that important IMO.

part of #189675 and #189486

### CODE REVIEW

File: src/vs/workbench/contrib/accessibility/browser/accessibleView.ts [hunk 1]
```diff
@@ -365,17 +365,21 @@ class AccessibleView extends Disposable {
...
+			this._accessiblityHelpIsShown.set(true);
+			const delegate: IContextViewDelegate = {
+				getAnchor: () => { return { x: (window.innerWidth / 2) - ((Math.min(this._layoutService.dimension.width * 0.62 /* golden cut */, DIMENSIONS.MAX_WIDTH)) / 2), y: this._layoutService.offset.quickPickTop }; },
+				render: (container) => {
+					container.classList.add('accessible-view-container');
+					return this._render(accessibleViewHelpProvider, container, true);
+				},
+			};
+			this._contextViewService.showContextView(delegate);
+		}, 100);
```
Comment (Tyriar) [inline src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:382]: Can 0 (or no time) work here? That will put the execution on the next task, if that doesn't work then you can't guarantee this won't introduce a race condition.

**Review by Tyriar — DISMISSED** [review]

File: src/vs/workbench/contrib/accessibility/browser/accessibleView.ts (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: 0 does not work here. based on my testing, 50 ms works fine for this and other such instances.

Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: it's not ideal, but I'm not sure of a better solution after spending a lot of time trying to come up with one

Comment (Tyriar) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: Do you have a link to what this is fixing? We don't want to just add a hack that we need to fix later, is there some event we could listen to other than an arbitrary time?

Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: fixing the same issue as #186514

the focused element changes, but the screen reader (voice over) does not detect that focus change if we don't wait a little bit

Comment (Tyriar) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: Does hiding the context view remove the element or just hide it?

Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: Just hides it. Maybe I should try disposing of it?

Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:384]: tried removing the element and I don't think that's what we want to do here

Comment (Tyriar) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:360]: If the context menu doesn't remove it, that might be the cause of the issue?

Comment (meganrogge) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:360]: that's how the context view service works and I don't see other things removing the element

Comment (Tyriar) [reply src/vs/workbench/contrib/accessibility/browser/accessibleView.ts:360]: Maybe the combination of removing and `setTimeout(, 0)` will do it? Shame if not, but let's add a `// HACK: desc...` comment to make it clear what's happening

**Review by roblourens — APPROVED** [review]

**Pull request merged**